### ISSUE TITLE
Latest update renders Nearby Stops inaccessible to visually impaired users

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/onebusaway-ios#525]
Erin Fitzpatrick writes:

> Hello,
> 
> I work with blind and DeafBlind transit users - the most recent update made the nearby stops feature inaccessible with voiceover. Please let us know if this can be fixed and when that might happen.
> 
> Erin Fitzpatrick
> Supervisor of Orientation and Mobility, Seattle Location
> The Lighthouse for the Blind, Inc.

**Comment by ualch9** [comment]
User demonstrated issue to me over Zoom.

I believe the root cause is two separate problems: #544 is interfering with the currently focused element, and OBAListView's background view (empty data view) should not be focusable when it is not visible (screenshot).
![IMG_0004](https://user-images.githubusercontent.com/22162410/136090436-d0255246-0990-4b36-a5d9-8488036a8daf.PNG)

User demonstrated on what appeared to be either iPhone 11 Pro or 12 Pro. Curiously, I tested this on an iPhone 11 Pro and an iPhone 13 Pro, and only the iPhone 11 had this problem.

**Comment by aaronbrethorst** [comment]
great work @ualch9
